src/clr-angular/utils/component/parse-boolean-attribute.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export function parseBooleanAttribute(value: string | boolean | number): boolean {
```

Comment:
We don't need this function, plus it duplicates what is in the other boolean attribute helper function.